src/Compiler/Utilities/Cancellable.fs

```diff
@@ -2,47 +2,37 @@ namespace FSharp.Compiler
 
 open System
 open System.Threading
-open Internal.Utilities.Library
 
 [<Sealed>]
 type Cancellable =
     [<ThreadStatic; DefaultValue>]
-    static val mutable private tokens: CancellationToken list
+    static val mutable private token: CancellationToken option
```

Comment:
Shall we use CancellationToken.None instead?

Reply:
Looking at this again today, I also think that's the way to go, thanks!